core/src/main/java/org/stellar/anchor/sep24/Sep24TransactionBuilder.java

```diff
@@ -48,6 +48,11 @@ public Sep24TransactionBuilder stellarAccount(String stellarAccount) {
     return this;
   }
 
+  public Sep24TransactionBuilder stellarAccountMemo(String accountMemo) {
+    txn.setAccountMemo(accountMemo);
+    return this;
+  }
```

Comment:
💡  These names are inconsistent. Throughout other parts of the project, we use just `memo` and `memoType`.

Reply:
This is actually the SEP-10 account memo which is different from the `memo` (transaction memo).

Reply:
This is not from SEP. It is from Polaris when I ported from Python to Java. 

Changing the name will break the MGI's implementation. I will add comments to the fields to avoid confusion.